src/components/Login/Login.js

```diff
@@ -179,7 +184,11 @@ class Login extends Component {
     if (error) {
       this.alert('Sever error(Profile)! Please sign in again.');
     } else if (profile) {
-      Actions.generalInfo({ me: profile });
+      if (profile.status === 200) {
```

Comment:
To solve sequential problem, I added it
@SarahhhC said that code 200 is for old user and 201 is for 201.